### PR TITLE
Issue25: fix incorrect settings.json configuration usage

### PR DESCRIPTION
**Pull request by rngadam**
Issue #25: fix incorrect settings.json configuration usage

### CODE REVIEW

File: app/.meteor/packages [hunk 1]
```diff
@@ -4,40 +4,43 @@
...
 app-root
-autopublish
+autopublish@1.0.7
 nimble:restivus
 kadira:blaze-layout
 kadira:flow-router
 useraccounts:flow-routing
 aldeed:collection2
 sacha:spin
+peerlibrary:stacktrace
```
Comment (thimovss) [inline app/.meteor/packages]: remove

File: app/.meteor/versions [hunk 2]
```diff
@@ -1,104 +1,111 @@
...
-reactive-var@1.0.6
-reload@1.1.4
-retry@1.0.4
-routepolicy@1.0.6
+npm-mongo@1.5.48
+oauth@1.1.11
+oauth2@1.1.10
+observe-sequence@1.0.12
+ordered-dict@1.0.8
+peerlibrary:stacktrace@0.2.0
```
Comment (thimovss) [inline app/.meteor/versions]: remove

File: app/server/init.js [hunk 3]
```diff
@@ -58,12 +62,14 @@ function initOrgs() {
 
 Meteor.startup(function () {
   var config = getCurrentConfiguration();
-
+  console.log(JSON.stringify(config));
```
Comment (thimovss) [inline app/server/init.js]: JSON stringify not needed (maybe)

File: app/server/login.js [hunk 4]
```diff
@@ -4,13 +4,15 @@ Accounts.onCreateUser(function(options, user) {
     console.log('error creating user');
     return;
   } else {
-    console.log(options, user)
```
Comment (thimovss) [inline app/server/login.js:7]: restore

File: app/server/login.js [hunk 5]
```diff
@@ -4,13 +4,15 @@ Accounts.onCreateUser(function(options, user) {
...
     user.profile = options.profile;
 
     if(user.services.github) {
       user.profile.email = user.services.github.email;
-      user.emails = user.services.github.emails;
```
Comment (thimovss) [inline app/server/login.js:12]: fix this

File: app/.meteor/packages (hunk 1 above)
Comment (rngadam) [reply app/.meteor/packages]: fixed

File: app/.meteor/versions (hunk 2 above)
Comment (rngadam) [reply app/.meteor/versions]: fixed

File: app/server/init.js (hunk 3 above)
Comment (rngadam) [reply app/server/init.js]: fixed (only work up to a certain depth)

File: app/server/login.js (hunk 4 above)
Comment (rngadam) [reply app/server/login.js:7]: fixed

File: app/server/login.js (hunk 5 above)
Comment (rngadam) [reply app/server/login.js:12]: fixed

**Pull request merged**